MAUI/Picker/overview.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: About .NET MAUI Picker control | Syncfusion
+description: Learn here about the overview of Syncfusion .NET MAUI Picker (SfPicker) control, its basic features, and picker functionalities.
+platform: maui
+control: SfPicker
+documentation: ug
+---
+
+# Overview of .NET MAUI Picker (SfPicker)
+
+Syncfusion .NET MAUI Picker (SfPicker) control allows you to select an item and visualizes its items inside a popup or a drop-down UI element. It supports multiple-column layout, header, footer, and selection view customizations. Privides customization as a date picker, time picker, date-time picker, country picker, color picker, etc.
```

Comment:
Syncfusion .NET MAUI Picker (SfPicker) control allows you to select an item and visualize its items inside a popup or a drop-down UI element. It supports multiple-column layout, header, footer, and selection view customizations. It provides customization as a date picker, time picker, date-time picker, country picker, color picker, and more.

Reply:
Content modified.